config/routes.rb

```diff
@@ -1,11 +1,17 @@
 Rails.application.routes.draw do
   namespace :api do
     namespace :v1 do
+
+      # USER SPECIFIC ROUTES
       resources :users, except: [:new] do
+        resources :shipments, only: [:index, :show], controller: :users_shipments
```

Comment:
wanted to namespace this but need it to not have redundant names in the route, i'll need to clean this up later